src/pages/docs/platform/pricing/limits.mdx

```diff
@@ -109,6 +109,7 @@ Integration limits relate to the rate and concurrency of [webhooks](/docs/integr
 | ----------------- | ---- | -------- | --- | ---------- |
 | **Integration rate (per second)**<p>*the maximum rate at which integrations can be executed, or streamed*</p> | 250 | 250 | 1,000 | Custom |
 | **Integration concurrency (Google Functions and Azure)**<p>*the maximum number of integrations that can run at the same time*</p> | 25 | 25 | 100 | Custom |
+| **Webhook concurrency**<p>*the maximum number of webhook requests that can be processed at the same time*</p> | 25 | 25 | 100 | Custom |
```

Comment:
> the maximum number of webhook requests that can be processed at the same time

Not sure this captures it. It implies we're processing something, but the whole point with this limit is that it's the _customer_ processing it. Maybe "The maximum number of unbatched webhook requests that can be in-flight at any one time from a given source"?

Also I'm not sure I understand the reasoning for presenting this one limit as two different ones.

(and indeed the one above is probably out of date since the limit only applies to http1 unbatched webhooks but google functions and azure endpoints probably accept http2 these days)